pg_shard--1.0--1.1.sql

```diff
@@ -12,7 +12,16 @@ AS $sync_table_metadata_to_citus$
 		table_relation_id CONSTANT oid NOT NULL := table_name::regclass::oid;
 		dummy_shard_length CONSTANT bigint := 0;
 	BEGIN
-		-- copy shard placement metadata
+		-- grab lock for upsert
+		LOCK TABLE pg_dist_shard_placement IN EXCLUSIVE MODE;
+
+		-- perform update of shard health
+		UPDATE pg_dist_shard_placement
+		SET    shardstate = shard_placement.shard_state
+		FROM   pgs_distribution_metadata.shard_placement
+		WHERE  shardid = shard_placement.shard_id;
+
+		-- copy new shard placement metadata
 		INSERT INTO pg_dist_shard_placement
```

Comment:
We use 90 characters as the limit on everything except git commit messages… I think we also use tabs (four columns) everywhere to be consistent. We don't really have an SQL style specified, so I used [this little web page](http://www.dpriver.com/pp/sqlformat.htm) to format things in v1.0 just so they'd have some sort of consistency without us really having to think much about it.

I'll look into whether there are places I can join lines to make them longer, but I want to stick with the output of this tool for now, since previous checkins used its style. That way we can keep line noise/churn down.